src/views/virtualmachines/details/tabs/configuration/storage/components/tables/disk/DiskList.tsx

```diff
@@ -35,9 +39,30 @@ const DiskList: FC<DiskListProps> = ({ customize = false, onDiskUpdate, vm, vmi
   const { createModal } = useModal();
   const columns = useDiskColumns();
   const [disks, sourcesLoaded, loadError] = useDisksTableData(vm, vmi);
+
   const filters = useDisksFilters();
   const [data, filteredData, onFilterChange] = useListPageFilter(disks, filters);
 
+  const accessReviewAttributes = useMemo(
+    () => [
+      {
+        group: VirtualMachineModel.apiGroup,
+        namespace: getNamespace(vm),
+        resource: VirtualMachineModel.plural,
+        verb: 'create' as K8sVerb,
+      },
+      {
+        group: VirtualMachineModel.apiGroup,
+        namespace: getNamespace(vm),
+        resource: VirtualMachineModel.plural,
+        verb: 'update' as K8sVerb,
+      },
+    ],
+    [vm],
+  );
+
+  const [results] = useMultipleAccessReviews(accessReviewAttributes);
```

Comment:
please don't use this function as its not multicluster. use `useFleetAccessReview`  for `create` DataVolumeModel.plural and `update` VirtualMachineModel.plural.

Name the two variables `canCreateDataVolume` and `canUpdate` and pass both into the DiskSourceSelect.